src/fides/api/models/identity_salt.py

```diff
@@ -0,0 +1,42 @@
+from sqlalchemy import Boolean, CheckConstraint, Column, UniqueConstraint
+from sqlalchemy.ext.declarative import declared_attr
+from sqlalchemy.ext.mutable import MutableDict
+from sqlalchemy_utils.types.encrypted.encrypted_type import (
+    AesGcmEngine,
+    StringEncryptedType,
+)
+
+from fides.api.db.base_class import Base  # type: ignore[attr-defined]
+from fides.api.db.base_class import JSONTypeOverride
+from fides.config import CONFIG
+
+
+class IdentitySalt(Base):
+    """
+    A single-row table used to store the encrypted salt value used for identity hashing (SHA-256).
```

Comment:
I decided to go with this automatic approach to prevent the user from having to deal with new Fides config settings. I also know the salt isn't strictly a sensitive value, but anything extra we do to hide it won't hurt.